### ISSUE TITLE
Uncaught Error: ENOENT: no such file or directory, lstat '/atom:'

### ISSUE THREAD

**Issue by moleksyuk** [atom/fuzzy-finder#191]
[Enter steps to reproduce below:]

1. ...
2. ...

**Atom Version**: 1.7.0
**System**: Mac OS X 10.11.4
**Thrown From**: [fuzzy-finder](https://github.com/atom/fuzzy-finder) package, v1.0.4

### Stack Trace

Uncaught Error: ENOENT: no such file or directory, lstat '/atom:'

```
At ATOM_SHELL_ASAR.js:0

Error: ENOENT: no such file or directory, lstat '/atom:'
    at Error (native)
    at fs.lstatSync (fs.js:887:18)
    at Object.fs.lstatSync (ATOM_SHELL_ASAR.js:229:16)
    at Object.realpathSync (fs.js:1532:21)
    at Object.fs.realpathSync (ATOM_SHELL_ASAR.js:304:29)
    at /Applications/Atom.app/Contents/Resources/app.asar/node_modules/fuzzy-finder/lib/path-loader.js:19:21
    at Array.map (native)
    at Object.module.exports.startTask (/Applications/Atom.app/Contents/Resources/app.asar/node_modules/fuzzy-finder/lib/path-loader.js:17:46)
    at ProjectView.module.exports.ProjectView.runLoadPathsTask (/Applications/Atom.app/Contents/Resources/app.asar/node_modules/fuzzy-finder/lib/project-view.js:194:46)
    at ProjectView.module.exports.ProjectView.populate (/Applications/Atom.app/Contents/Resources/app.asar/node_modules/fuzzy-finder/lib/project-view.js:111:21)
    at ProjectView.module.exports.ProjectView.toggle (/Applications/Atom.app/Contents/Resources/app.asar/node_modules/fuzzy-finder/lib/project-view.js:87:14)
    at atom-workspace.atom.commands.add.fuzzy-finder:toggle-file-finder (/Applications/Atom.app/Contents/Resources/app.asar/node_modules/fuzzy-finder/lib/main.js:9:46)
    at CommandRegistry.module.exports.CommandRegistry.handleCommandEvent (/Applications/Atom.app/Contents/Resources/app.asar/src/command-registry.js:260:29)
    at /Applications/Atom.app/Contents/Resources/app.asar/src/command-registry.js:3:61
    at KeymapManager.module.exports.KeymapManager.dispatchCommandEvent (/Applications/Atom.app/Contents/Resources/app.asar/node_modules/atom-keymap/lib/keymap-manager.js:580:16)
    at KeymapManager.module.exports.KeymapManager.handleKeyboardEvent (/Applications/Atom.app/Contents/Resources/app.asar/node_modules/atom-keymap/lib/keymap-manager.js:388:22)
    at WindowEventHandler.module.exports.WindowEventHandler.handleDocumentKeyEvent (/Applications/Atom.app/Contents/Resources/app.asar/src/window-event-handler.js:98:36)
    at HTMLDocument.<anonymous> (/Applications/Atom.app/Contents/Resources/app.asar/src/window-event-handler.js:3:61)
```

### Commands

```
 10x -5:23.6.0 core:backspace (atom-text-editor.editor.is-focused)
     -3:04.3.0 core:select-all (atom-text-editor.editor.is-focused)
     -3:02.9.0 core:copy (atom-text-editor.editor.is-focused)
 12x -3:02.4.0 core:paste (atom-text-editor.editor.is-focused)
     -2:59 core:select-all (atom-text-editor.editor.is-focused)
     -2:58.2.0 core:backspace (atom-text-editor.editor.is-focused)
     -2:34.2.0 command-palette:toggle (ul.list-inline.tab-bar.inset-panel)
     -2:31.4.0 editor:consolidate-selections (atom-text-editor.editor.mini.is-focused)
     -2:31.4.0 core:cancel (atom-text-editor.editor.mini.is-focused)
  2x -1:21.2.0 core:select-all (atom-text-editor.editor.is-focused)
     -0:49.3.0 application:open-your-stylesheet (atom-workspace.workspace.scrollbars-visible-when-scrolling.theme-one-dark-syntax.theme-one-dark-ui)
  2x -0:30 core:select-all (atom-text-editor.editor.is-focused)
     -0:00.7.0 fuzzy-finder:toggle-file-finder (ol.tree-view.full-menu.list-tree.has-collapsable-children.focusable-panel)
```

### Config

```json
{
  "core": {
    "audioBeep": false,
    "themes": [
      "one-light-ui",
      "one-light-syntax"
    ]
  }
}
```

### Installed Packages

```coffee
# User
No installed packages

# Dev
No dev packages
```

**Comment by Tibicenas** [comment]
+1  👍 


▶ atom -v
Atom    : 1.7.2
Electron: 0.36.8
Chrome  : 47.0.2526.110
Node    : 5.1.1

OSX El Capitán 10.11.4

**Comment by lee-dohm** [comment]
Can you give some indication of when or how this happens?

**Comment by moleksyuk** [comment]
Hi @lee-dohm 
I've just installed it and tried to open some file as I remember.
It was for the first time I tried Atom, so I can't exactly provide steps to reproduce it.

**Comment by 50Wliu** [comment]
@moleksyuk Did you by any chance move the Atom.app file (or similar, I'm not too familiar with how Atom is packaged on Mac) after starting it?

**Comment by moleksyuk** [comment]
Hi @50Wliu 
No I didn't. Just installed it to /Application folder and run it.

**Comment by Tibicenas** [comment]
same for me

**Comment by draeath** [comment]
Same, but mine was installed via (homebrew)[http://brew.sh/]

My files are located in `/opt/homebrew-cask/Caskroom/atom/1.7.2/Atom.app` and a symbolic link exists as `~/Applications/Atom.app` to launch it. If I press ⌘t to pop up the fuzzy finder, this error is presented.

```
Atom    : 1.7.2
Electron: 0.36.8
Chrome  : 47.0.2526.110
Node    : 5.1.1
```

**Comment by walshp7595** [comment]
I just hit this. Flag said upgrade would occur at next start of Atom; I restarted it, entered ⌘P and got the error. I am running Mac OS X 10.10.5. Restarting Atom and issuing the same keystrokes does not give error, so for me too looks like first-time-only occurrence..

**Comment by alias1** [comment]
Fresh install of atom 1.7.3 via homebrew on OSX El Capitan 10.11.3

Hit this when I pressed ⌘P, only happened the first time.

**Comment by SekharKolli** [comment]
Just hit this now...Since I'm new to atom, listing all that I remember, perhaps you might what you are looking for
1. installed a fresh copy of atom (never had it on my mac)
2. I opened a folder to edit a index.html page pulled from github and made a few changes to it.
3. Mac was in hibernate (closed the screen with atom and the file running) and opened it after 12+ hrs 
4. Choose to explore/install a few packages. Think that last that I installed was the jQuery-snippets package successfully. I also did have a few panes open.
5. Choose to close all open files. The operation completed I think. However my index.html file was still visible to me on screen.
6. Thinking that I did not close the file correctly, I choose to close it again, this time using the command pallet. So I  hit the `shift+alt+p` shortcut (which you know is an incorrect shortcut, no command pallet appeared, and something could have happened in the background).
7. Now I hit the correct shortcut key immediately after step 6 `shift+cmd+p`, typed 'close all' enter, and the crash.

**Comment by 50Wliu** [comment]
Fixed by #204.

**Comment by schwaz80** [comment]
Just received this same error and I'm on 1.8
`[Enter steps to reproduce below:]

1. ...
2. ...

**Atom Version**: 1.8.0
**System**: Mac OS X 10.11.3
**Thrown From**: [fuzzy-finder](https://github.com/atom/fuzzy-finder) package, v1.0.5


### Stack Trace

Uncaught Error: ENOENT: no such file or directory, lstat '/atom:'

```
At ATOM_SHELL_ASAR.js:0

Error: ENOENT: no such file or directory, lstat '/atom:'
    at Error (native)
    at fs.lstatSync (fs.js:887:18)
    at Object.fs.lstatSync (ATOM_SHELL_ASAR.js:229:16)
    at Object.realpathSync (fs.js:1532:21)
    at Object.fs.realpathSync (ATOM_SHELL_ASAR.js:304:29)
    at /Applications/Atom.app/Contents/Resources/app.asar/node_modules/fuzzy-finder/lib/path-loader.js:19:21
    at Array.map (native)
    at Object.module.exports.startTask (/Applications/Atom.app/Contents/Resources/app.asar/node_modules/fuzzy-finder/lib/path-loader.js:17:46)
    at ProjectView.module.exports.ProjectView.runLoadPathsTask (/Applications/Atom.app/Contents/Resources/app.asar/node_modules/fuzzy-finder/lib/project-view.js:190:46)
    at ProjectView.module.exports.ProjectView.populate (/Applications/Atom.app/Contents/Resources/app.asar/node_modules/fuzzy-finder/lib/project-view.js:111:21)
    at ProjectView.module.exports.ProjectView.toggle (/Applications/Atom.app/Contents/Resources/app.asar/node_modules/fuzzy-finder/lib/project-view.js:87:14)
    at atom-workspace.atom.commands.add.fuzzy-finder:toggle-file-finder (/Applications/Atom.app/Contents/Resources/app.asar/node_modules/fuzzy-finder/lib/main.js:9:46)
    at CommandRegistry.module.exports.CommandRegistry.handleCommandEvent (/Applications/Atom.app/Contents/Resources/app.asar/src/command-registry.js:260:29)
    at /Applications/Atom.app/Contents/Resources/app.asar/src/command-registry.js:3:61
    at KeymapManager.module.exports.KeymapManager.dispatchCommandEvent (/Applications/Atom.app/Contents/Resources/app.asar/node_modules/atom-keymap/lib/keymap-manager.js:580:16)
    at KeymapManager.module.exports.KeymapManager.handleKeyboardEvent (/Applications/Atom.app/Contents/Resources/app.asar/node_modules/atom-keymap/lib/keymap-manager.js:388:22)
    at WindowEventHandler.module.exports.WindowEventHandler.handleDocumentKeyEvent (/Applications/Atom.app/Contents/Resources/app.asar/src/window-event-handler.js:98:36)
    at HTMLDocument.<anonymous> (/Applications/Atom.app/Contents/Resources/app.asar/src/window-event-handler.js:3:61)
```

### Commands

```
     -1:19.5.0 core:close (atom-text-editor.editor.is-focused)
     -1:07.8.0 find-and-replace:show (atom-text-editor.editor.is-focused)
  3x -1:06.0 core:move-up (atom-text-editor.editor.mini.is-focused)
  3x -1:05.2.0 core:move-down (atom-text-editor.editor.mini.is-focused)
  4x -1:04.5.0 core:move-up (atom-text-editor.editor.mini.is-focused)
     -1:02.9.0 core:move-down (atom-text-editor.editor.mini.is-focused)
     -0:32.5.0 find-and-replace:find-all (atom-text-editor.editor.mini.is-focused)
     -0:28.2.0 core:copy (atom-text-editor.editor.is-focused)
     -0:04.0 fuzzy-finder:toggle-file-finder (atom-text-editor.editor.is-focused)
```

### Config

```json
{
  "core": {}
}
```

### Installed Packages

```coffee
# User
No installed packages

# Dev
No dev packages
```

`
